csm/core/services/audit_log.py

```diff
@@ -132,7 +132,6 @@ def is_visible(field_id):
             ["user_agent", "User agent", 801],
             ["response_code", "Response code", 901],
             ["request_id", "Request ID", 1001],
-            ["msg", "Message", 1101],
```

Comment:
I think there was a defect in S3 Auditlog due to which we had added it as a fallback mechanism and we added it in CSM too.
@soniyamoholkar  can you please check and confirm